tests/pytests/unit/states/test_boto_cloudfront.py

```diff
@@ -30,7 +30,7 @@ def configure_loader_modules(setup_vars):
         whitelist=[
             "boto3",
             "dictdiffer",
-            "yaml",
+            "yamldumper",
```

Comment:
**question** if I do `git checkout salt/master -- salt/loader/lazy.py` then this doesn't fail :thinking: It only fails if I leave lazy as-is and change this back. Why is that?

Reply:
Because now the loader, correctly, only adds functions defined in the module, not functions imported into the module, unless these are namespaced into the module scope.